microsoft-edge/toc.yml

```diff
@@ -351,6 +351,12 @@
               href: extensions-chromium/publish/manage-settings.md
             - name: Contact extensions support
               href: extensions-chromium/publish/contact-extensions-team.md
+            - name: Microsoft Edge Add-ons Publish API
```

Comment:
Need to look at the rendered TOC (and top-of-page titles) and decide if this is good wording and placement.
https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/api/using-addons-publish-api?branch=user%2Fv-mhoffman%2Faddons-publish-api
By design, a TOC title is shorter than the top-of-page title.